dev/com.ibm.ws.microprofile.health/bnd.bnd

```diff
@@ -38,7 +38,7 @@ Import-Package: \
   *
 
 Export-Package: \
-    com.ibm.ws.microprofile.health.resources; version="1.0", \
+    !com.ibm.ws.microprofile.health.resources,\
```

Comment:
Should it be added to the Private-Package list?

Reply:
The `com.ibm.ws.microprofile.health.resources` was already in the Private-Package list.

Reply:
I even looked twice and didn't see it.  I see it now.  I must be blind today.  :)